idr/jsonnode.go

```diff
@@ -1,33 +1,73 @@
 package idr
 
+// JSONType is the type of a JSON-specific Node.
+// Note multiple JSONType can be bit-wise OR'ed together.
 type JSONType uint
 
 const (
+	// JSONRoot is the type for the root Node in a JSON IDR tree.
 	JSONRoot JSONType = 1 << iota
+	// JSONRoot is the type for a Node in a JSON IDR tree whose value is an object.
```

Comment:
here the comment seems should be `JSONObj`

Reply:
fixed